arch/arm/core/aarch32/Kconfig

```diff
@@ -21,6 +21,7 @@ config CPU_CORTEX_M
 	select ARCH_HAS_EXTRA_EXCEPTION_INFO
 	select ARCH_HAS_TIMING_FUNCTIONS if CPU_CORTEX_M_HAS_DWT
 	select ARCH_SUPPORTS_ARCH_HW_INIT
+	select ARCH_HAS_SUSPEND_TO_RAM if PM
```

Comment:
Shouldn't this select be unconditional?
The arch either supports it or not, and `CONFIG_PM_S2RAM` is already dependant on `CONFIG_PM` being selected.

Reply:
right, I'll change it.